### ISSUE TITLE
Batch timing needs to be spread out and balanced to avoid batching spikes

### ISSUE THREAD

**Issue by brick-green-agile6** [CDCgov/prime-reportstream#3941]
## Problem statement

76 out of 91 receivers have timing starting at 1:15am or 9:15am ET every day. This results in a batching spike at 09:15 with pagerduty alerts sent out shortly after.

## To do

- [ ] Develop plan to balance batch timing
- [ ] Adjust production settings

**Comment by anshulkumar-usds** [comment]
Need to determine if this is needed, and plan out usage across receivers.

**Comment by anshulkumar-usds** [comment]
@brick-green-agile6 please set time with Pipeline and DevOps on next steps.

**Comment by brick-green-agile6** [comment]
Here is the breakdown of our current batch timing:
![image.png](https://images.zenhubusercontent.com/60d23d7dcaf07751be9205b6/51e77bf4-44b5-496e-8bc8-cb8858ceb371)
![image.png](https://images.zenhubusercontent.com/60d23d7dcaf07751be9205b6/1fae84f9-01ac-424a-a915-3c698ebcd098)

Proposed batch timing:
![image.png](https://images.zenhubusercontent.com/60d23d7dcaf07751be9205b6/c782fec6-ff5e-4bac-b90e-490e08108864)
![image.png](https://images.zenhubusercontent.com/60d23d7dcaf07751be9205b6/bc099e68-1b69-4f41-9b56-8726822ab2e8)

**Comment by brick-green-agile6** [comment]
@Adrian-Brewster is there any communication that will be needed here to the STLTs? I tried to keep the file creation time change to under 20 mins but a few have a change of 35 mins. In my mind that still fairly small and I can't see that being an issue for any STLTs. But what do you think?

**Comment by Adrian-Brewster** [comment]
@brick-green-agile6 No, I dont think so.  When discussing batch timing in the past, there were no states that I can think of that had specific requirements and often it is discussed in a "around this time" sort of way. We should be good

**Comment by brick-green-agile6** [comment]
cool, working the change now

**Comment by brick-green-agile6** [comment]
PR was merged. Now I need to make the changes in staging and prod
